x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_item.tsx

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
This file will be removed in the next iteration. All link configs will be taken from the new general `appLinks` configs and formatted to the `NavItem` & `PortalNavItem` type.